Make.config

```diff
@@ -117,6 +117,7 @@ endif
 ## If this branch is a release branch, set NUGET_RELEASE_BRANCH to the exact branch name (so that any other branches won't become release branches just by branching off from a release branch).
 ## Example: release/6.0.3xx
 # NUGET_RELEASE_BRANCH=
+NUGET_HARDCODED_IS_STABLE_BUILD=false
```

Comment:
@rolfbjarne @mandel-macaque I'm not sure if this is the best place to put this. I am not too familiar with the changes you need to make when updating packages for stable versioning/branding, but I believe most/all of those changes are made to this file?

Is there some already existing variable or property that we could check instead, so that we don't add another item that needs to be manually updated whenever moving to stable package branding?

Reply:
Since both are out, we can have this as is for now for p7. @pjcollins could you please file an issue so we do not forget?

Reply:
We can actually hold this for a bit since it will only impact stable branded packs, I'll fix the conflict here

Reply:
Updated this to use NUGET_PRERELEASE_IDENTIFIER as suggested